code-scanning/zscaler-iac-scan.yml

```diff
@@ -0,0 +1,52 @@
+#This workflow uses actions that are not certified by GitHub.
+#They are provided by a third party and are governed by
+#separate terms of service, privacy policy, and support
+#documentation.
+
+#This workflow runs the Zscaler Infrastructure as Code (IaC) Scan app,
+#which detects security misconfigurations in IaC templates and publishes the findings
+#under the code scanning alerts section within the repository.
```

Comment:
Add few more details if possible:
- Any pre-requisites
- Any link to learn more
- Any link to get support or create issue